api/graphql/schema.graphql

```diff
@@ -495,7 +495,7 @@ type Team implements HasRoleAssignments {
   roleAssignments: [RoleAssignment!]
     @hasMany
     @canRoot(ability: "viewTeamMembers")
-  teamIdForRoleAssignment: ID @rename(attribute: "id")
+  teamIdForRoleAssignment: ID @rename(attribute: "id") @with(relation: "team")
 }
```

Comment:
This breaks it, there is no team relation on team

![image](https://github.com/user-attachments/assets/80fa0037-8bdb-4d0e-9ef6-e7205d8bfe72)

Reply:
Huh, that makes sense. Kind of confusing that it implements the interface then :face_with_spiral_eyes: 

Anyway, removed in [remove overeager loading of teams](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11404/commits/8311cd5e745eb94b0c763ab24f38c6ce596287f6)

Reply:
Yeah, the teams stuff is in a bit of a confusing place right now due to being midway through the shift to communities

Reply:
:100: